ratis-shell/pom.xml

```diff
@@ -24,9 +24,45 @@
   <name>Apache Ratis Shell</name>
 
   <properties>
+    <guava.version>29.0-jre</guava.version>
   </properties>
 
   <dependencies>
+    <dependency>
+      <groupId>com.google.guava</groupId>
+      <artifactId>guava</artifactId>
+      <version>${guava.version}</version>
+      <exclusions>
+        <exclusion>
+          <artifactId>jsr305</artifactId>
+          <groupId>com.google.code.findbugs</groupId>
+        </exclusion>
+      </exclusions>
+    </dependency>
+    <dependency>
+      <groupId>commons-cli</groupId>
+      <artifactId>commons-cli</artifactId>
+      <version>1.3.1</version>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.commons</groupId>
+      <artifactId>commons-lang3</artifactId>
+      <version>3.5</version>
+    </dependency>
```

Comment:
Only StringUtils is used and it can be easily replaced by some other code.  Let's don't add common-lang3.

Reply:
Nice suggestion! I use `String.join(" ", argv),` to replace `StringUtils.join`